packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -120,12 +112,12 @@ export class OrgList implements vscode.Disposable {
     return authList;
   }
 
-  public async updateOrgList() {
-    const authInfoObjects = await this.getAuthInfoObjects();
-    if (!authInfoObjects) {
+  public async updateOrgList(): Promise<string[] | null> {
+    const orgAuthorizations = await this.getOrgAuthorizations();
+    if (orgAuthorizations && orgAuthorizations.length === 0) {
       return null;
```

Comment:
nitty thing, but here you could return an empty array. Then the return type here can be just string[]; This simplifies lines 134-136 removing the need to check if authInfoList exists.

Reply:
Updated!